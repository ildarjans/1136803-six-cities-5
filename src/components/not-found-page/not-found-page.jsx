import React from "react";
import {Link} from "react-router-dom";
import {Header} from "../header/header";

export const NotFoundPage = () => {
  return (
    <>
      <Header/>
      <h1>Page not Found</h1>
      <br/>
      <Link to='/'>Go to main page</Link>
    </>
  );
};
