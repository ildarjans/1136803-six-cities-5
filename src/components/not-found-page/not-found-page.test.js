import React from "react";
import renderer from "react-test-renderer";
import {BrowserRouter} from "react-router-dom";
import {NotFoundPage} from "./not-found-page";

jest.mock(`../header/header`);

it(`Should NotFoundPage rendered correctly`, () => {
  const tree = renderer.create(
      <BrowserRouter>
        <NotFoundPage/>
      </BrowserRouter>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
