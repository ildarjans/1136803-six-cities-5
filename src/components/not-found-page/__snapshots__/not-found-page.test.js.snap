// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should NotFoundPage rendered correctly 1`] = `
Array [
  <h1>
    Page not Found
  </h1>,
  <br />,
  <a
    href="/"
    onClick={[Function]}
  >
    Go to main page
  </a>,
]
`;
